src/common/language_eeprom.cpp

```diff
@@ -0,0 +1,46 @@
+
+#include "language_eeprom.hpp"
+#include "../lang/translator.hpp"
+
+/*!
+ * Setter & getter class to store language code into a EEPROM
+ * Constructor will check if language code is not set, then it will set
+ * a default value.
+ * If it's set, then Translator is called to start translating.
+ */
+LangEEPROM::LangEEPROM()
+    : _language(0) {
+    uint16_t _language = static_cast<uint16_t>(eeprom_get_var(EEVAR_LANGUAGE).ui16);
+    if (_language == static_cast<uint16_t>(0xffff)) {
+        setLanguage(Translations::MakeLangCode("cs"));
+    } else {
+        Translations::Instance().ChangeLanguage(_language);
```

Comment:
I'm not sure if the LangEEPROM class should do anything else than reading and saving the lang code - i.e. calling Translations::Instance here looks like an unnecessary dependency. I'd rather see this in the Translations class - explicitly obtaining the correct lang code from LangEEPROM

Reply:
This class after discussion will stay as a manager class with Translator dependency to change newly set language code right after it's stored into a EEPROM. This will avoid unneccessary explicit calling getting language and then call change language to translator.